mmv1/products/compute/ForwardingRule.yaml

```diff
@@ -92,6 +92,18 @@ examples:
       network_name: 'website-net'
     ignore_read_extra:
       - 'port_range'
+  - name: 'forwarding_rule_externallb_byoipv6'
+    primary_resource_id: 'default'
+    min_version: 'beta'
+    vars:
+      forwarding_rule_name: 'test-byoipv6-forwarding-rule'
+      backend_name: 'website-backend'
+      network_name: 'website-net'
+      ip_address: '2600:1901:4457:1::/96'
+    test_vars_overrides:
+      ip_address: 'fmt.Sprintf("2600:1901:4457:1:%d:%d::/96", rand.Intn(9999), rand.Intn(9999))'
```

Comment:
I thought we could just use the `rand` package but apparently it's a bit more complicated than that - you'll need to add a new `acctest.RandIntn` function and use that. You can add the function to the standard test utils after RandInt https://github.com/GoogleCloudPlatform/magic-modules/blob/de41d5d077af282f1bfde22f2c1a6a136c3e96aa/mmv1/third_party/terraform/acctest/test_utils.go.tmpl#L123